crates/precompile/src/secp256r1.rs

```diff
@@ -77,70 +77,15 @@
 //!     }
 //! }
 //! ```
-use crate::addresses::P256VERIFY_ADDRESS;
+//use crate::addresses::P256VERIFY_ADDRESS;
```

Comment:
we can remove this entire file, and therefore we can remove the entire `odyssey-precompile` crate

Reply:
yes